std/c/dragonfly.zig

```diff
@@ -0,0 +1,2884 @@
+pub const INT_FAST32_MIN = INT32_MIN;
+pub const KERN_MAXID = 37;
+pub const KERN_PROC_PID = 1;
+pub const CTL_P1003_1B = 9;
+pub const KERN_VERSION = 4;
+pub const CTLTYPE_S8 = 12;
+pub const CTL_P1003_1B_MEMLOCK = 3;
+pub const KERN_BOOTTIME = 21;
+pub const __UINT_FAST32_MAX__ = c_uint(4294967295);
+pub const CTL_P1003_1B_MEMLOCK_RANGE = 4;
+pub const OID_AUTO = -1;
+pub const CTL_P1003_1B_ASYNCHRONOUS_IO = 1;
+pub const CTL_P1003_1B_AIO_PRIO_DELTA_MAX = 17;
+pub const KERN_PROC_TTY = 4;
+pub const CTL_DEBUG = 5;
+pub const USER_POSIX2_VERSION = 10;
+pub const USER_POSIX2_SW_DEV = 17;
+pub const CTLFLAG_RD = 2147483648;
+pub const KERN_VNODE = 13;
+pub const KERN_FILE = 15;
+pub const USER_POSIX2_FORT_RUN = 15;
+pub const HW_PHYSMEM = 5;
+pub const CTLTYPE_ULONG = 8;
+pub const USER_POSIX2_C_DEV = 12;
+pub const KERN_OSRELDATE = 24;
+pub const KERN_MAXPROCPERUID = 28;
+pub const __INT_FAST64_FMTd__ = c"ld";
+pub const CTLTYPE_STRING = 3;
+pub const USER_POSIX2_LOCALEDEF = 16;
+pub const KERN_PROF = 16;
+pub const CTLFLAG_DYING = 65536;
+pub const KIPC_MAXSOCKBUF = 1;
+pub const __SSE2_MATH__ = 1;
+pub const CTL_MAXID = 11;
+pub const KERN_SAVED_IDS = 20;
+pub const KIPC_MAX_LINKHDR = 4;
+pub const KERN_SECURELVL = 9;
+pub const KIPC_MBSTAT = 8;
+pub const HW_MACHINE = 1;
+pub const KERN_PROC_SIGTRAMP = 10;
+pub const CTL_P1003_1B_SEM_VALUE_MAX = 23;
+pub const __AVX__ = 1;
+pub const KERN_PROC = 14;
+pub const KERN_PROC_FLAG_LWP = 16;
+pub const USER_BC_DIM_MAX = 3;
+pub const CTL_P1003_1B_PRIORITIZED_IO = 7;
+pub const HW_USERMEM = 6;
+pub const KERN_PROC_UID = 5;
+pub const __DBL_MAX_EXP__ = 1024;
+pub const CTL_P1003_1B_MESSAGE_PASSING = 6;
+pub const KIPC_MAX_PROTOHDR = 5;
+pub const CTL_P1003_1B_SEMAPHORES = 10;
+pub const __SSE3__ = 1;
+pub const CTL_HW = 6;
+pub const CTL_P1003_1B_SYNCHRONIZED_IO = 13;
+pub const KERN_CLOCKRATE = 12;
+pub const __clang_patchlevel__ = 0;
+pub const KERN_JOB_CONTROL = 19;
+pub const CTLTYPE = 15;
+pub const CTLTYPE_QUAD = CTLTYPE_S64;
+pub const HW_MAXID = 14;
+pub const HW_DISKSTATS = 9;
+pub const CTL_P1003_1B_PRIORITY_SCHEDULING = 8;
+pub const CTL_UNSPEC = 0;
+pub const CTLFLAG_DYN = 33554432;
+pub const KERN_ARGMAX = 8;
+pub const KERN_PROC_RUID = 6;
+pub const HW_SENSORS = 13;
+pub const CTL_VM = 2;
+pub const CTL_NET = 4;
+pub const CTL_P1003_1B_DELAYTIMER_MAX = 18;
+pub const CTL_P1003_1B_REALTIME_SIGNALS = 9;
+pub const CTLTYPE_STRUCT = CTLTYPE_OPAQUE;
+pub const __UINTMAX_FMTx__ = c"lx";
+pub const CTLFLAG_SECURE = 134217728;
+pub const KERN_PROC_CWD = 8;
+pub const CTLTYPE_S32 = 14;
+pub const CTL_VFS = 3;
+pub const CTLFLAG_SKIP = 16777216;
+pub const TRASHIT = x;
+pub const KERN_HOSTNAME = 10;
+pub const USER_LINE_MAX = 8;
+pub const CTLFLAG_PRISON = 67108864;
+pub const CTLFLAG_SHLOCK = 32768;
+pub const KERN_PROC_ALL = 0;
+pub const KERN_PROC_ARGS = 7;
+pub const CTLTYPE_S16 = 13;
+pub const CTLTYPE_NODE = 1;
+pub const __PTRDIFF_FMTi__ = c"li";
+pub const __SSE4_1__ = 1;
```

Comment:
this is about *your* computer; not from dragonfly itself.

Reply:
:flushed:  Thanks @daurnimator :wink:

Reply:
Is this better?